packages/cli/src/commands/start/watchMode.ts

```diff
@@ -39,7 +39,7 @@ function enableWatchMode(messageSocket: any) {
           process.exit();
           break;
         case 'z':
-          process.emit('SIGTSTP');
+          process.emit('SIGTSTP', 'SIGTSTP');
```

Comment:
Curious why these two (same) arguments are required? What they're supposed to do?

Reply:
It looks like process emit is based on the Events emitter https://nodejs.org/api/events.html#events_emitter_emit_eventname_args and the second argument is passed to the user defined listener/handler, but I'm not 100% sure about this. 😉

Reply:
I see. We could've used it then to pass some useful data but I don't think we have a use case for it right now.